homeworks/05-css-advanced/styles.css

```diff
@@ -293,87 +293,111 @@ span,
 
 .main__image-p {
     text-align: center;
-    font-size: 0.875rem;
+    font-size: 14px;
 }
 
 .main__image-button {
-    width: 2.375rem;
-    height: 2.375rem;
+    width: 38px;
+    height: 38px;
     border: none;
     outline: none;
     border-radius: 50%;
     cursor: pointer;
 }
 
 .main__image-button svg {
-    width: 1.125rem;
-    height: 0.625rem;
+    width: 18px;
+    height: 10px;
 }
 
 .footer__p {
-    margin: 6rem 0 3rem;
+    padding: 60px 0 40px 0;
 }
 
 .scroll {
     display: none;
 }
 
-@media (max-width: 1024px) {
+
+@media (max-width: 600px) {
 
     html {
-        font-size: 12px;
+        font-size: 8px;
     }
 
-    .header__button svg {
-        width: 2.2rem;
-        height: 2.2rem;
+    .header__button {
+        width: 48px;
+        height: 48px;
     }
 
-    .main__img {
-        width: 330px;
+    .header__button svg {
+        width: 20px;
+        height: 20px;
     }
 
-}
+    .header__main-img {
+        width: 107px;
+        height: 107px;
+    }
 
-@media (max-width: 600px) {
+    .header__title {
+        font-size: 32px;
+    }
 
-    html {
-        font-size: 8px;
+    .header__subtitle,
+    .section__stats-p,
+    .main__portfolio-text,
+    .main__skills,
+    .footer__p {
+        font-size: 14px;
     }
 
-    .header__button svg {
-        width: 2.5rem;
-        height: 2.5rem;
+    .header__title,
+    .header__subtitle {
+        text-align: center;
     }
 
     .header__main {
         flex-direction: column;
     }
 
     .header__main-link svg {
-        width: 2.5rem;
-        height: 2.5rem;
+        width: 20px;
+        height: 20px;
     }
 
     .header__main-developer {
         padding: 0;
     }
 
-    .header__title,
-    .header__subtitle {
-        text-align: center;
-    }
-
     .header__main-list {
         justify-content: center;
         margin-top: 2rem;
     }
 
+    .section__stats-title {
+        font-size: 16px;
+    }
+
+    .section__secondary {
+        margin-top: 30px;
+        margin-bottom: 30px;
```

Comment:
Not all sizes according to design and CSS Box Model.
For example here: 
why did you added mb 30px???
It is child element - for shifting it from parent's border we use padding;  for creating a gap to the next element (.main) you should have had to add marginBottom to parrent(.section) and even in this case this indent would be redundant, because .main has margin-top 4rem for this purpose.